Help.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Report</title>
+        <link rel="stylesheet" href="style.css"  type="text/css">
+    </head>
+
+    <body>
+        <nav>
+            <ul>
+                <li><a><img src="https://cdn.discordapp.com/attachments/755129724811870211/757711994370392211/unknown.png" class="logo" class="menu"></a></li>
+                <li><a href="index.html" class="menu">Home</a></li>
+                <li><a href="Browse.html" class="menu">Browse</a></li>
+                <li><a href="Help.html" class="menu">Help</a></li>
+                <li><a href="https://forms.gle/QmQRTCUmqyDopNDF7" class="menu">Report</a></li>
+                <button type="button" id="theme_changer" class="button">Change theme</button>
+            </ul>
+        </nav>
+
+        <p>This is the help page</p>
+        <p>You can navigate to other parts of the website by pressing the menu buttons at the top on the navigation bar</p>
+        <p>The home page is just for show</p>
+        <p>The browse page is where you can browse the archived videos/pictures</p>
+        <p>The feed back page sends you to a feedback form that you can send us feedback</p>

```

Comment:
Using the word "feedback" three times in a sentence might be redundant.
```suggestion
        <p>The feedback page sends you to a form where you can send us feedback</p>
```